bin/gitmeta.jl

```diff
@@ -105,7 +106,9 @@ function gitmeta(pkgs::Dict{String,Package})
             git_tree_hash = string(LibGit2.GitHash(git_tree))
             d[uuid][v.sha1] = git_tree_hash
             # scan for stdlib dependencies
-            (v"0.7" in v.requires["julia"].versions || v"1.0" in v.requires["julia"].versions) || continue
+            v"0.7" in v.requires["julia"].versions ||
```

Comment:
Maybe this should be updated to `any(v -> v >= v"0.7", v.requires["julia"].versions)` or something?

Reply:
Sure, that could work too. Just wanted to get it fixed quickly since it was blocking packages working.